src/ReactLeafletMap/index.js

```diff
@@ -70,86 +87,113 @@ class ReactLeafletMap extends React.PureComponent {
         }
     }
 
-    componentDidMount() {
-        // Hack for ugly 1px tile borders in Chrome
-        // Version of Leaflet package in dependencies should match version used by react-leaflet
-        let originalInitTile = L.GridLayer.prototype._initTile;
-        L.GridLayer.include({
-            _initTile: function (tile) {
-                originalInitTile.call(this, tile);
+    setZoomLevelsBounds(width, height) {
```

Comment:
Should this be called on props change?

Reply:
It should. However, in react-leaflet has the Map component so-called dynamic and static props and minZoom and maxZoom are for some reason static. Therefore this is called on initial resize only.